main.go

```diff
@@ -79,10 +80,23 @@ func main() {
 		os.Exit(1)
 	}
 
+	ctx := ctrl.SetupSignalHandler()
+
+	kubeConfig, err := ctrl.GetConfig()
+	if err != nil {
+		setupLog.Error(err, "unable to get kubeConfig", "controller", "SearchOperator")
+		os.Exit(1)
+	}
+	addonMgr, err := addon.NewAddonManager(kubeConfig)
+	if err != nil {
+		setupLog.Error(err, "unable to create a new  addon manager", "controller", "SearchOperator")
+		os.Exit(1)
+	}
```

Comment:
I don't want to exit on these errors because this operator does other functions.  These errors create a significant degradation, but we could still perform the other tasks.

Reply:
done

Reply:
thanks!